megamek/src/megamek/common/util/ImageUtil.java

```diff
@@ -211,10 +207,13 @@ public Image loadImage(String fileName) {
                 return null;
             }
             Image result = Toolkit.getDefaultToolkit().getImage(fileName);
-            if(null == result) {
+            if (result == null) {
+                return null;
+            }
+            final boolean isAnimated = waitUntilLoaded(result);
+            if ((result.getWidth(null) == -1) || (result.getHeight(null) == -1)) {
```

Comment:
I'd recommend < 0

Reply:
The choice was based on the JavaDoc comment in Image::getWidth and Image::getHeight, but it cannot be less than 0 so that's a better way to handle it. Flipped over.